helm/values-dev.yaml

```
@@ -179,6 +179,8 @@ microservice-chart:
     NPG_AUTHORIZATION_EXCLUDED_ERROR_CODES: "GW0035,GW0004"
     EXCLUSIVE_LOCK_DOCUMENT_TTL_SECONDS: "2"
   envSecret:
+    NODO_CLOSEPAYMENT_API_KEY: nodo-nodeforpm-api-key
```

```suggestion
    NODO_NODEFORPM_API_KEY: nodo-nodeforpm-api-key
```

use the same naming convention for both api keys. please refactor application property key too

---

done